### ISSUE TITLE
Cant access kubeflow dashboard after using "kubectl port-forward svc/istio-ingressgateway -n istio-system --address 0.0.0.0 8085:80"

### ISSUE THREAD

**Issue by TranThanh96** [awslabs/kubeflow-manifests#332]
I tried install kubeflow on aws with s3 storage by following tutorial from https://awslabs.github.io/kubeflow-manifests/docs/deployment/
Everything works well except the last step to access kubeflow dashboard: kubectl port-forward svc/istio-ingressgateway -n istio-system --address 0.0.0.0 8085:80

After using port-fowarding, I cant access http://localhost:8080/
This page gave me 403 error: You don't have authorization to view this page!
How can I fix  this?

**Comment by AlexandreBrown** [comment]
@TranThanh96 Can you make sure your command is the same as the doc https://awslabs.github.io/kubeflow-manifests/docs/deployment/vanilla/guide/#port-forward ?  
```shell
kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
```

**Comment by ryansteakley** [comment]
Hey @TranThanh96, I responded to you on slack, can you additionally specify which deployment option you ran, was it the rds-s3?

**Comment by TranThanh96** [comment]
> @TranThanh96 Can you make sure your command is the same as the doc https://awslabs.github.io/kubeflow-manifests/docs/deployment/vanilla/guide/#port-forward ?
> 
> ```shell
> kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
> ```

ya, I tried both with and without  --address 0.0.0.0
still cant access from browser

**Comment by TranThanh96** [comment]
> Hey @TranThanh96, I responded to you on slack, can you additionally specify which deployment option you ran, was it the rds-s3?

I use s3 only

**Comment by TranThanh96** [comment]
after re-installing everything, I can reach the login page now

**Comment by ryansteakley** [comment]
sounds good, verify you are able to login and run any samples you wish.

**Comment by TranThanh96** [comment]
> 



> sounds good, verify you are able to login and run any samples you wish.

@ryansteakley I cant see any example pipelines in dashboard
![image](https://user-images.githubusercontent.com/26323599/186332540-999331f6-a65d-4cf6-a303-7a1bd770cbad.png)

and I cant create a new notebook server, error: 0/1 nodes are available : 1 too many pods

**Comment by ryansteakley** [comment]
looks like you have several pods in crashloop backoff. Is your instance the same size or similar to the one described in https://awslabs.github.io/kubeflow-manifests/docs/deployment/prerequisites/ Did you follow the auto-setup python script?

**Comment by ryansteakley** [comment]
run kubectl describe pod <pod-name> -n <namespace> and similarily kubectl logs <pod-node> -n <namespace> on the pods in failure state. and share anything you find there as well

**Comment by TranThanh96** [comment]
@ryansteakley 
I follow guide install with s3 only.
I am using 2 node: t3.xlarge
now I have 3 pod that keep CrashLoopBackOff: 

![image](https://user-images.githubusercontent.com/26323599/186358245-5d75dd81-6812-4d5c-8eb0-2d933f9d7399.png)


and this is log for each pod
[metadata-grpc-deployment-f8d68f687-pdzcx_describe.txt](https://github.com/awslabs/kubeflow-manifests/files/9413877/metadata-grpc-deployment-f8d68f687-pdzcx_describe.txt)
[metadata-grpc-deployment-f8d68f687-pdzcx_log..txt](https://github.com/awslabs/kubeflow-manifests/files/9413878/metadata-grpc-deployment-f8d68f687-pdzcx_log.txt)
[metadata-writer-d7ff8d4bc-qqtjz_describe.txt](https://github.com/awslabs/kubeflow-manifests/files/9413879/metadata-writer-d7ff8d4bc-qqtjz_describe.txt)
[metadata-writer-d7ff8d4bc-qqtjz_log.txt](https://github.com/awslabs/kubeflow-manifests/files/9413880/metadata-writer-d7ff8d4bc-qqtjz_log.txt)
[ml-pipeline-777648985d-jhkvl_describe.txt](https://github.com/awslabs/kubeflow-manifests/files/9413881/ml-pipeline-777648985d-jhkvl_describe.txt)
[ml-pipeline-777648985d-jhkvl_log.txt](https://github.com/awslabs/kubeflow-manifests/files/9413882/ml-pipeline-777648985d-jhkvl_log.txt)

**Comment by ryansteakley** [comment]
`Warning  Failed     34m (x5 over 34m)      kubelet            Error: secret "mlpipeline-minio-artifact" not found` in ml-pipeline logs. Can you check to see if this secret exists. Run `kubectl get secrets -n kubeflow`

**Comment by TranThanh96** [comment]
[secrets_kf_log.txt](https://github.com/awslabs/kubeflow-manifests/files/9414439/secrets_kf_log.txt)
seem like it exist
![image](https://user-images.githubusercontent.com/26323599/186373090-4500b24b-240c-4866-b636-44aa4b3ef52a.png)

**Comment by ryansteakley** [comment]
Can you verify that you are using v3.2.0 of kustomize? Run `kubectl delete pods -n kubeflow --all` and see if the pods come up normally.

**Comment by TranThanh96** [comment]
yes, I am using kustomize v3.2.0
![image](https://user-images.githubusercontent.com/26323599/186378091-a805cefe-053f-4331-9fce-559132958812.png)

Tried kubectl delete pods -n kubeflow --all but the pod metadata-grpc-deployment-f8d68f687-mqs82  keep crashloopbackoff

![image](https://user-images.githubusercontent.com/26323599/186378284-74733df1-15d0-4c31-9d25-b073bb8aceaf.png)

**Comment by ryansteakley** [comment]
What do you see when you login? Are any other pods still failing?

**Comment by TranThanh96** [comment]
everything is good except those 3 pods keep crashloopbackoff
![image](https://user-images.githubusercontent.com/26323599/186387250-a4a2580d-1913-4fdb-a43d-1ed05f57ab14.png)

and I get some errors on Pipelines and Runs, any suggestions please?

![image](https://user-images.githubusercontent.com/26323599/186387496-02205744-8945-4b14-b683-419164ba1905.png)
![image](https://user-images.githubusercontent.com/26323599/186387521-4c50ac5d-e0c7-4af8-9189-fe02824fdff4.png)


errors on Runs:
![image](https://user-images.githubusercontent.com/26323599/186387594-45f185da-c68e-48fc-b4d3-4516a8084547.png)
![image](https://user-images.githubusercontent.com/26323599/186387625-750dea2f-3bb2-4b45-a943-385a7d832e3f.png)

and these pods:
![image](https://user-images.githubusercontent.com/26323599/186387691-a18a1681-089e-4510-a601-760e469df3e2.png)

**Comment by ryansteakley** [comment]
Can you verify that the s3-secret you created is following this requirement.  `Configure a Secret (e.g. s3-secret) with your AWS credentials. These need to be long-term credentials from an IAM user and not temporary.`

**Comment by TranThanh96** [comment]
> 

yes, I can confirm that

**Comment by ryansteakley** [comment]
No way, to prove. Can you one more time describe the ml-pipeline pod. I would suggest restarting from a fresh cluster, and follow the cluster pre-req listed above.

**Comment by TranThanh96** [comment]
> 

Yes, this is 3rd times I re-install kubeflow on aws eks from a fresh cluster. and this error keep occurring

**Comment by ryansteakley** [comment]
Sorry you are running into these problems, if you can please share the logs from the latest crashloopbackoff mlpipeline. Which version of AWS kubeflow are you running? I will try to reproduce your issue on my end and see if there is some underlying issue.

**Comment by TranThanh96** [comment]
> Sorry you are running into these problems, if you can please share the logs from the latest crashloopbackoff mlpipeline. Which version of AWS kubeflow are you running? I will try to reproduce your issue on my end and see if there is some underlying issue.

how can I get these log? I can provide it to you.
I am using this version:
KUBEFLOW_RELEASE_VERSION=v1.5.1
AWS_RELEASE_VERSION=v1.5.1-aws-b1.0.1

**Comment by ryansteakley** [comment]
`kubectl logs <ml-pipeline-pod> -n kubeflow` i see you are running 2 node: t3.xlarge, we reccomend a minimum of 5 nodes and m5.xlarge. Stated here https://awslabs.github.io/kubeflow-manifests/docs/deployment/prerequisites/ if you have time try to re-create following the suggested cluster create command

**Comment by TranThanh96** [comment]
> kubectl logs <ml-pipeline-pod> -n kubeflow

![image](https://user-images.githubusercontent.com/26323599/186393025-f5059e01-4a43-4919-9227-cafcf3c545a6.png)
This is log from ml-pipeline

**Comment by surajkota** [comment]
@ryansteakley @TranThanh96 I think this is because of a bug related to missing mysql deployment in S3 only deployment option. It was fixed in `main` branch recently but not backported to release branch https://github.com/awslabs/kubeflow-manifests/pull/310

@TranThanh96 Can you comment out this like `- disable-mysql-pv-claim.yaml` in `awsconfigs/apps/pipeline/s3/kustomization.yaml` and run

`kustomize build awsconfigs/apps/pipeline/s3 | kubectl apply -f -`

Please delete the ml-pipeline and grpc pod after doing this so that a new pod gets created

**Comment by TranThanh96** [comment]
> @ryansteakley @TranThanh96 I think this is because of a bug related to missing mysql deployment in S3 only deployment option. It was fixed in `main` branch recently but not backported to release branch #310
> 
> @TranThanh96 Can you comment out this like `- disable-mysql-pv-claim.yaml` in `awsconfigs/apps/pipeline/s3/kustomization.yaml` and run
> 
> `kustomize build awsconfigs/apps/pipeline/s3 | kubectl apply -f -`
> 
> Please delete the pods which are in crashloopbackoff after doing this so that a new pod gets created

yes, I try with rds + s3 deployment. everything works. So the problem is related to  mysql

**Comment by surajkota** [comment]
Thanks for reporting this issue. We have released a patch version (v1.5.1-aws-b1.0.2) to fix this issue
